engine/execution/testutil/fixtures.go

```diff
@@ -304,25 +303,67 @@ func CreateAccountCreationTransaction(t *testing.T, chain flow.Chain) (flow.Acco
 	return accountKey, tx
 }
 
+// CreateAddAnAccountKeyMultipleTimesTransaction generates a tx that adds a key several times to an account.
```

Comment:
Maybe a comment for the next person trying to use this ("this can be used to exhaust an account's storage") ?

Reply:
added that.